examples/Injection/README.md

```diff
@@ -0,0 +1,39 @@
+# Injection scenario
+
```

Comment:
Maybe a short description of the scenario so it is clear what to expect without going through the trouble of actually executing the whole thing. Wasn't this used in some publication? If yes, cite it here.

Reply:
Thanks for your review! I've just implemented your suggestions.